src/IssueViz/Editor/QuickActions/QuickFixes/QuickFixSuggestedAction.cs

```diff
@@ -28,11 +29,21 @@ internal class QuickFixSuggestedAction : BaseSuggestedAction
     {
         private readonly IQuickFixVisualization quickFixVisualization;
         private readonly ITextBuffer textBuffer;
+        private readonly ISpanTranslator spanTranslator;
 
-        public QuickFixSuggestedAction(IQuickFixVisualization quickFixVisualization, ITextBuffer textBuffer)
+        public QuickFixSuggestedAction(IQuickFixVisualization quickFixVisualization,
+            ITextBuffer textBuffer)
+            : this(quickFixVisualization, textBuffer, new SpanTranslator())
+        {
+
+        }
+        internal QuickFixSuggestedAction(IQuickFixVisualization quickFixVisualization, 
```

Comment:
We have a team convention that any classes that are helper/utility are initialized in an internal ctor (that the tests also call), and the public ctor passes a default value.